### ISSUE TITLE
Add new assay to a `QFeatures` object when calling `impute()`

### ISSUE THREAD

**Issue by cvanderaa** [lgatto/Features#162]
Currently, `impute()` replaces missing data with imputed data. I would prefer that `impute()` adds a new assay. This will make comparison between imputation methods easier. 

Here is a dummy example. Instead of: 
```r
library("QFeatures")
data("feat1")
impute(feat1, i = "psms", method = "zero")
```
```
An instance of class QFeatures containing 1 assays:
 [1] psms: SummarizedExperiment with 10 rows and 2 columns 
```

I would suggest to have:

```r
data("feat1")
impute(feat1, i = "psms", name = "psms_impd", method = "zero")
```
```
An instance of class QFeatures containing 2 assays:
 [1] psms: SummarizedExperiment with 10 rows and 2 columns 
 [2] psms_impd: SummarizedExperiment with 10 rows and 2 columns 
```

Let me know your thoughts. If you agree, I can assign myself to it.

**Comment by lgatto** [comment]
Yes, definitely 👍🏻

**Comment by cvanderaa** [comment]
Solved by #163
